package.json

```diff
@@ -4,14 +4,18 @@
   "description": "Atomic Flux with hot reloading",
   "main": "lib/index.js",
   "scripts": {
-    "browser": "scripts/browser",
-    "build": "scripts/build",
-    "clean": "scripts/clean",
-    "lint": "scripts/lint",
-    "prepublish": "scripts/prepublish",
-    "test": "scripts/test",
-    "test:watch": "scripts/test-watch",
-    "test:cov": "scripts/test-cov"
+    "build": "npm run clean && babel src --out-dir lib",
+    "clean": "rimraf lib dist coverage",
+    "lint": "eslint src test examples",
+    "prepublish": "npm run build",
+    "release": "mversion patch -m",
+    "release:patch": "mversion patch -m",
+    "release:minor": "mversion minor -m",
+    "release:major": "mversion major -m",
+    "test": "npm run lint && NODE_ENV=test mocha --compilers js:babel/register --recursive",
+    "test:watch": "npm test -- --watch",
+    "test:cov": "npm run lint && babel-node ./node_modules/.bin/isparta cover ./node_modules/.bin/_mocha -- --recursive",
+    "umd": "npm run clean && npm run lint && webpack src/index.js dist/redux.js && NODE_ENV=production webpack src/index.js dist/redux.min.js"
```

Comment:
I don't like that we do `clean` and `lint` in most commands. Isn't it better to have a few low-level “focused” commands that do just one thing (e.g. call `webpack`) and then assemble higher-level commands from them?

For example right now linting and cleaning will happen twice if I build both lib & UMD.